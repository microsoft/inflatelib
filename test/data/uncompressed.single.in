
# This generates a binary file with a single uncompressed block. The file consists of the bytes 0-255, in order

>1
1       # bfinal = true since this is the only block
00      # No compression

# NOTE: This will byte align, which is desired
>16
0100    # LEN = 256
FEFF    # NLEN = ~LEN

# And finally, the data
>8
00...FF
