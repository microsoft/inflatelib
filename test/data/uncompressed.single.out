
# Corresponds to 'uncompressed.single.in'. The output is just the bytes 0-255
00...FF
